images/sonic/launch.py

```diff
@@ -226,6 +227,17 @@ def get_ip_address(iface: str) -> str:
     )[20:24])
 
 
+def get_netmask(iface: str) -> str:
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    # Use ioctl to get the netmask
+    netmask = socket.inet_ntoa(fcntl.ioctl(
+        s.fileno(),
+        0x891b,  # SIOCGIFNETMASK
+        struct.pack('256s', iface.encode('utf-8'))
+    )[20:24])
```

Comment:
This is for sure not working in the ipv6 case

Reply:
I never thought about using IPv6 for the management interface. Should I look into it?

Reply:
No, some sort of clarification above would help for people not familiar with ioctl, and pydoc that says that this method only works for ipv4